src/analysis/Environments/UnsupportedWorkspaceCell.tsx

```diff
@@ -1,18 +1,20 @@
 import { TooltipTrigger } from '@terra-ui-packages/components';
-import { div, h } from 'react-hyperscript-helpers';
+import { useThemeFromContext } from '@terra-ui-packages/components';
```

Comment:
The two imports from `@terra-ui-packages/components` should be combined in one line.